code/src/components/Form.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Form = ({ newThought, setNewThought, formSubmit }) => {
+  return (
+    <div className="form-wrapper">
+      <form onSubmit={formSubmit}>
+        <label htmlFor="newThought">What's making you happy right now?</label>
+        <div>
+          <textarea
```

Comment:
Interesting alternative, nice to see that it also works with textarea (I used input).